src/NuGet.Clients/NuGet.PackageManagement.UI/Resources.resx

```diff
@@ -927,4 +927,13 @@ Please see https://aka.ms/troubleshoot_nuget_cache for more help.</value>
   <data name="ArgumentNullOrEmpty" xml:space="preserve">
     <value>Argument cannot be null or empty.</value>
   </data>
+  <data name="Accessibility_ColumnNotSortedHelpText" xml:space="preserve">
+    <value>Not sorted, press space to sort ascending</value>
```

Comment:
Are we 100% sure it's right to say "space" here? I see this string is copied form at least one other VS area but just feels wrong to me to announce a key.

Reply:
Our keys are "space" or "enter" and in order to simplify what gets read but I added only one of them as guidance to reduce noise. This is what the Error List does as well and that's been around for a bit so I based it on that. I've been told that non-sighted users will already be familiar with what keys to press to sort column headers and the guidance is not strictly necessary but the bug filed suggests that guidance should be provided so I've added it here.

Reply:
I think the bug is more about indicating sorting is available and state, not which key to press. It's unusual but if it's the VS status quo, for 16.9 there's no need to challenge it.